stubs/sublime_plugin.pyi

```diff
@@ -1,27 +1,35 @@
 import sublime
-from typing import Optional, Union, List, Dict, Tuple
 
+from typing import Callable, Generic, TypeVar, Optional, Union, List, Tuple, overload
 
-class CommandInputHandler():
+
+InputType = TypeVar('InputType')
```

Comment:
We could limit this to something like a `Union[str, int, float, list, dict, tuple, None]`, since it must be serializable, but optional for me.

Reply:
Fixed.